openQA.spec

```diff
@@ -229,7 +229,14 @@ rm -f t/00-tidy.t
 #make test
 rm -rf %{buildroot}/DB
 export LC_ALL=en_US.UTF-8
-make test-with-database OBS_RUN=1 PROVE_ARGS='-l -r -v' TEST_PG_PATH=%{buildroot}/DB || true
+# Skip tests not working currently, or flaky, and Selenium tests
+# https://progress.opensuse.org/issues/19652
+rm \
```

Comment:
FYI the same should be possible to achieve with 
```
make test-with-database OBS_RUN=1 PROVE_ARGS='-l -r -v' TEST_PG_PATH=%{buildroot}/DB GLOB_IGNORE='t/25-cache-service.t:t/ui/*.t'
```

Reply:
thanks

Reply:
One reasn I'd like to keep it like that is that it gets kind of unreadable.
We also remove `t/00-tidy.t` above, so that should be added here also, and it's hard to find if you actually wantto check which tests are skipped.

The team seems to favor long lines, though, so if you want, I can do that change.

Reply:
As far as I'm concerned it's not long lines but a) rather exclude than delete b) a single command can easily be copied for testing - you could still use line breaks and the variable

Reply:
For what it's worth I wouldn't worry about changing it, though. It's good either way